test/common_distributed.py

```diff
@@ -146,8 +150,21 @@ def _join_processes(self, fn):
         for p in self.processes:
             p.join(timeout)
         elapsed_time = time.time() - start_time
-        if fn in self.skip_return_code_checks:
+        if fn not in self.skip_return_code_checks:
             self._check_return_codes(elapsed_time)
```

Comment:
What about the other exit statuses where you want to skip status checking?

Reply:
Nevermind, I thought this was intentional before, but it was an error before and is now fixed.